licenses.d2iq.yaml

```
@@ -143,6 +147,14 @@ resources:
     sources:
       - url: https://github.com/mesosphere/dkp-insights
         ref: main
+  - container_image: docker.io/mesosphere/insights:v0.2.3
+    sources:
+      - url: https://github.com/mesosphere/dkp-insights
+        ref: v0.2.3
+  - container_image: docker.io/mesosphere/insights:v0.4.1
+    sources:
+      - url: https://github.com/mesosphere/dkp-insights
+        ref: v0.4.1
```

i think we can use `${image_tag}` here if the image tag matches the tag on the repo (which looks like it does)
```suggestion
        ref: ${image_tag}
```